packages/mobile/src/account/GoldEducation.tsx

```diff
@@ -51,27 +55,19 @@ function useStep() {
     return [
       {
         image: shinyGold,
-        cancelEvent: AnalyticsEvents.gold_cancel1,
-        progressEvent: AnalyticsEvents.gold_educate_1_next,
-        screenName: 'Gold_Nux_1',
+        topic: EducationTopic.celo,
```

Comment:
ditto, I must be missing something but how is it all 4 topics are the same value now?

Reply:
Logic is just moved to the `Education` screen where `topic` is the identifier